styles.css

```diff
@@ -36,6 +36,10 @@ nav h1 {
 	padding: 10px;
 }
 
+form {
+text-align: center;
+}
+
```

Comment:
Clean and tidy CSS. Could use 'padding:' with 4 values rather than use padding-top and padding-bottom for nav h1.